core/src/main/scala/org/apache/spark/sql/SnappyContext.scala

```diff
@@ -413,7 +413,7 @@ class SnappyContext protected[spark](val snappySession: SnappySession)
    * @param provider Provider name such as 'COLUMN', 'ROW', 'JDBC' etc.
    * @param schema   Table schema
    * @param options  Properties for table creation. See options list for different tables.
-   *              https://github.com/SnappyDataInc/snappydata/blob/master/docs/rowAndColumnTables.md
+   *              https://github.com/TIBCOSoftware/snappydata/blob/master/docs/rowAndColumnTables.md
```

Comment:
The filename does not exit anymore. Will need to find the alternative file and update here.
I think we have identified that in SDENT-196_new branch.